tools/ddm-declaration-items.sh

```diff
@@ -2,7 +2,12 @@
 
 URL="${BASE_URL}/declaration-items"
 
+if [ "x$API_USER" = "x" ]; then
+    API_USER="kmfddm"
+fi
+
 curl \
     $CURL_OPTS \
     -H "X-Enrollment-ID: $1" \
+    -u "$API_USER:$API_KEY" \
```

Comment:
```suggestion
    $CURL_AUTH
```

In this way, you can maintain a mode that does not require authentication.